README.rst

```diff
@@ -147,6 +147,29 @@ If you have external files you can add them in ``examples`` folder to skip.
 
 For rst-syntax-error skip unknown directives
 
+Skip one check based on the Odoo version
+--------------
+
+If you need to valid one check using the Odoo version you should be used the
+new variable ``odoo_check_versions``
+
+For example, if you have one new check `your-new-check` and you need run only
+if the Odoo version is between 9.0 and 11.0, inside the class you should be
```

Comment:
...you should add a declaration as follows...

Reply:
or ..declare the following...

Reply:
Thanks for review... fixed